UserInterface/ViewModels/MainWindowViewModel.cs

```diff
@@ -132,6 +172,36 @@ private void OnCurrentViewModelChanged()
         CurrentViewModel = navigationStore.CurrentViewModel;
     }
 
+    private void ChangeToBig(ResourceDictionary resourceDict)
+    {
+        UpdateResource(resourceDict, TextBlock.FontSizeProperty, "Text", 40.0);
+        UpdateResource(resourceDict, TextBlock.FontSizeProperty, "SubHeader", 48.0);
+        UpdateResource(resourceDict, TextBlock.FontSizeProperty, "Header", 56.0);
+    }
+    private void ChangeToSmall(ResourceDictionary resourceDict)
+    {
+        UpdateResource(resourceDict,TextBlock.FontSizeProperty, "Text", 32.0);
+        UpdateResource(resourceDict, TextBlock.FontSizeProperty, "SubHeader", 40.0);
+        UpdateResource(resourceDict, TextBlock.FontSizeProperty, "Header", 48.0);
+    }
+    private void UpdateResource(ResourceDictionary resourceDict, DependencyProperty Type, string field, double value) 
+    {
+        ((Style)resourceDict[field]).Setters.Add(new Setter(Type, value));
+    }
+    private void ChangeTextSize()
+    {
+        ResourceDictionary resourceDict = new ResourceDictionary { Source = new Uri("pack://application:,,,/UserInterface;component/Assets/Styling/TextStyles.xaml") };
+
+        if (!IsBigFontSize)
+            ChangeToBig(resourceDict);
+        else
+            ChangeToSmall(resourceDict);
+
+        Application.Current.Resources.MergedDictionaries.Add(resourceDict);
+
+        IsBigFontSize = !IsBigFontSize;
+    }
+
```

Comment:
Willen we elke keer een nieuwe resourcedictionary definiëren? Want blijft dat `Application.Current.Resources` in memory tijdens runtime?

Reply:
Ik heb gemerkt dat als ik er 1 aanmaak en iets aanpas je een error krijged dat hij gesealed is
dus ik weet niet of er een andere optie is.

Reply:
System.InvalidOperationException: 'After a 'SetterBaseCollection' is in use (sealed), it cannot be modified.'

Reply:
Hmm oke dan. Onze kennis gaat nog denk ik niet zo ver om hier wat mee te kunnen, I guess.... als het werkt dan werkt het :)

Reply:
alsjeblieft